documentation/concerns.md

```diff
@@ -44,7 +44,7 @@ This made up example shows how you can pass yourself information to help execute
 UserContext contextForUser = YourGraphqlContextBuilder.getContextForUser(getCurrentUser());
 
 ExecutionInput executionInput = ExecutionInput.newExecutionInput()
-        .context(contextForUser)
```

Comment:
I branched off the other documentation PR because this work touches on the same directives file.